styles/colors/base/light-hc.scss

```diff
@@ -0,0 +1,59 @@
+@mixin yc-colors-base-light-hc {
```

Comment:
can we import this file via new syntax here, like `@mixin colors`, and utilize namespaces like `@use './colors/base/light-hc' as lightHC;`? instead of giving some long name for mixin

Reply:
Well, if that's the way forward then I think all this imports should be changed together, maybe in a separate PR?

<img width="403" alt="Screen Shot 2022-08-31 at 23 07 10" src="https://user-images.githubusercontent.com/168296/187772330-f56a9372-4631-4871-b306-848ee69c01f8.png">